web/src/main/java/io/fabric8/launcher/web/api/OpenShiftResource.java

```diff
@@ -61,21 +61,23 @@
     @Inject
     private OpenShiftClusterRegistry clusterRegistry;
 
+    @Inject
+    private KeycloakService keycloakService;
```

Comment:
I changed it so you can, the check to see if the keycloakUrl is set is now moved from the constructor to the method that uses the keycloak url in order to use the keycloakService to get the identity in case there is no keycloak